lib/src/js/actions/get_token.js.dart

```diff
@@ -16,19 +16,25 @@ extension type JSGetTokenParameters._(JSObject _) implements JSObject {
 @JS()
 extension type JSGetTokenReturnType(JSObject _) implements JSObject {
   external JSString address;
-  external JSNumber decimals;
-  external JSString name;
-  external JSString symbol;
-  external JSAny totalSupply;
+  external JSNumber? decimals;
```

Comment:
According to https://wagmi.sh/core/api/actions/getToken, `decimals` is mandatory.